config/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/SIGBlockchain/project_aurum/internal/config"
+)
+
+func main() {
+	//open configuration file
+	configFile, err := os.Open("../cmd/config.json")
+	if err != nil {
+		log.Fatal("Failed to load configuration file : " + err.Error())
+	}
+	defer configFile.Close()
+
+	cfgData, err := ioutil.ReadAll(configFile)
+	if err != nil {
+		log.Fatal("Failed to read configuration file : " + err.Error())
+	}
+
+	cfg := config.Config{}
+	if err := json.Unmarshal(cfgData, &cfg); err != nil {
+		log.Fatal("Failed to unmarshall configuration data : " + err.Error())
+	}
+
+	//specify flags
+	versionU64 := flag.Uint64("version", uint64(cfg.Version), "enter version number")
```

Comment:
Not a big deal, but I think using a `flag.Uint` would be more appropriate than a 64-bit number, simply because we're eventually casting down to a 16-bit, so 32-bit might look better? idk

Reply:
I thought `flag.Uint` returns a pointer to a 8-bit uint? That's why I used the `Uint64` version...

Reply:
Integers are typically 32 bit. 

https://golang.org/pkg/builtin/#uint